miso-web/src/main/webapp/scripts/hot_qctype.js

```diff
@@ -0,0 +1,34 @@
+HotTarget.qctype = {
+  createUrl: '/miso/rest/qctype',
+  updateUrl: '/miso/rest/qctype/',
+  requestConfiguration: function(config, callback) {
+    callback(config)
+  },
+  fixUp: function(qctype, errorHandler) {
+  },
+  createColumns: function(config, create, data) {
+    return [
+      HotUtils.makeColumnForText('Name', true, 'name', {
+        unpackAfterSave: true,
+        validator: HotUtils.validator.requiredText
+      }), HotUtils.makeColumnForText('Description', true, 'description', {
+        unpackAfterSave: true,
+      }), HotUtils.makeColumnForEnum('Target', true, true, 'qcTarget', config.qcTargets, null),
```

Comment:
It shouldn't be possible to edit the target type for existing QCs. Add a check that looks at `create` and displays this column only when creating.